cloudconnexa/cloudconnexa.go

```diff
@@ -128,6 +128,7 @@ func (c *Client) DoRequest(req *http.Request) ([]byte, error) {
 
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.Token))
 	req.Header.Set("User-Agent", c.UserAgent)
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
Does this mean we no longer need [this line](https://github.com/OpenVPN/cloudconnexa-go-client/blob/main/cloudconnexa/cloudconnexa_test.go#L18) or [this line](https://github.com/OpenVPN/cloudconnexa-go-client/blob/main/cloudconnexa/cloudconnexa_test.go#L29)?

Reply:
Yes, removed setting Content-Type in tests